generators/client/files-angular.js

```diff
@@ -123,9 +123,6 @@ const files = {
         {
             path: ANGULAR_DIR,
             templates: [
-                // entities
-                'entities/_index.ts',
-                'entities/_entity.module.ts',
```

Comment:
I think we should still keep the entity module and import every other entity here so that the app.module remains small and cleaner